include/sfdo.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_SFDO_H
+#define LABWC_SFDO_H
+
+struct server;
+
+void sfdo_init(struct server *server);
+void sfdo_finish(struct server *server);
+struct lab_data_buffer *sfdo_icon_lookup(struct server *server,
+	const char *app_id, int size, float scale);
+const char *sfdo_desktop_entry_name_lockup(struct server *server,
```

Comment:
We can't (or rather: should not) use the `sfdo_` prefix because it belongs to libsfdo and might clash in the future.

Reply:
Ah yeah, good point. Do you have a suggestion? What comes to my mind is `desktop_entry` or `desktop_entries_db`, but I’m not convinced that this is the best fit.

Reply:
Can't come up with anything better than the `desktop_entry_` prefix either right now.

Reply:
In addition, I think we should add a comment about the lifetime of returned value. In this case, its lifetime will be the same as that of `sfdo_desktop_db`.

Also, there's a typo: `s/lockup/lookup/`.

Reply:
Thanks, addressed. I wasn't sure about the new name of the `struct icon_loader`. Using `desktop_entry` looks a bit weird, so I kept that as `sfdo`. Please let me know if you prefer `desktop_entry` of a different name instead.

Reply:
`struct sfdo` sounds fine to me.